docs/_config.yml

```
@@ -0,0 +1,114 @@
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.github_repo }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: gRPC-Gateway
+description: gRPC-Gateway Documentation Website
+baseurl: "/grpc-gateway" # the subpath of your site, e.g. /blog
+url: "https://grpc-ecosystem.github.io/grpc-gateway/" # the base hostname & protocol for your site, e.g. http://example.com
```

I think this should be 
```suggestion
url: "https://grpc-ecosystem.github.io/" # the base hostname & protocol for your site, e.g. http://example.com
```